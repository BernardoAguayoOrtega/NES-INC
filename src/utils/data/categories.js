// categories of the site
export const categories = [
  'Opinión',
  'Estados',
  'Cartones',
  'Mundo',
  'Empresas',
  'Colombia',
  'Internacionales',
  'Tecnología',
  'Gestión',
  'Deportes',
  'Judicial',
  'Entretenimiento',
  'Economía',
  'Arte e ideas',
  'Política',
  'Viral',
  'Santander',
  'Actualidad',
  'Autos',
  'Mercados'
]
