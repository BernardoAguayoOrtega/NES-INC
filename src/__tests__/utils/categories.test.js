// import app component
import { categories } from '../../utils/data/categories.js';

describe('request.js', () => {

  it('is match snapshot newsAPI?', () => {
    expect(categories)
      .toMatchSnapshot()
  });

});
