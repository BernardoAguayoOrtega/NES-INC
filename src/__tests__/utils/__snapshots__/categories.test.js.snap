// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`request.js is match snapshot newsAPI? 1`] = `
Array [
  "Opinión",
  "Estados",
  "Cartones",
  "Mundo",
  "Empresas",
  "Colombia",
  "Internacionales",
  "Tecnología",
  "Gestión",
  "Deportes",
  "Judicial",
  "Entretenimiento",
  "Economía",
  "Arte e ideas",
  "Política",
  "Viral",
  "Santander",
  "Actualidad",
  "Autos",
  "Mercados",
]
`;
